src/config.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Module for getting and setting Prebid configuration.
+ *
+ * Prebid previously defined these properties directly on the global object:
+ * pbjs.logging = true;
+ *
+ * Defining and access properties in this way is now deprecated, but these will
+ * continue to work during a deprecation window.
+ */
+
+const utils = require('./utils');
+
+const DEFAULT_DEBUG = false;
+const DEFAULT_BIDDER_TIMEOUT = 3000;
+
+export let config = {
```

Comment:
I think we need some way to subscribe to changes to this object.  For instance, the currency module (in review still) will need to kick off a request to go and download the currency file as soon as its configuration is set.  But without some sort of subscription model, it won't know when that is.